### DISCUSSION TITLE
Submission Error, please help check

### DISCUSSION THREAD

**Discussion by Jiangyi-Zhu** [PaddlePaddle/PaddleSpatial#222] [BaiduKDDCup2022] [answered]
Hi, the team name is zealen and the submission time is 2022-06-24 10:18. It appears that `Err: The preparation script, i.e. 'prepare.py', does NOT exist! ! (my_folder.zip)`. But I have included predict.py and prepare.py in the zipped file, could you please help check? Thanks

**Comment by ethanyxx** [comment, marked as answer]
Hi,
 
Below is the error log associated with your submission. 
![截屏2022-06-24 17 25 45](https://user-images.githubusercontent.com/62667125/175506567-3b5cec8d-639f-4798-a794-528c503993a6.png)

**Comment by panelding** [comment]
maybe I know the reason , you may submit you result zip file on Mac,  so in zip file there has some '__MAXOSX__' file.
at last the eval function in the evaluation.py return a false result dir.
![粘贴的图形-2](https://user-images.githubusercontent.com/31956407/176389681-07e0c4b3-8e8e-4034-90cf-e31f0e33c659.png)
